### PR TITLE
feat(setup): Halyard GCE quickstart

### PR DESCRIPTION
**Pull request by lwander**
I also move the GKE quickstart to the setup section, since it felt out-of-place and a little hard to find in under the user guides (which otherwise all deal with a running Spinnaker installation).

### CODE REVIEW

**Comment by lwander** [comment]
@EricJizbaMSFT FYI this moves the halyard quickstarts around

**Comment by lwander** [comment]
ping

File: setup/quickstart/halyard-gce.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+* Persistent storage configured to use GCS.
+* Spinnaker configured to deploy resources to GCE, and bake GCE VM images.
+
+## Part 0: Preparation
+
+### Install gcloud
+
+If you don't already have gcloud installed, navigate to [Installing Cloud SDK](https://cloud.google.com/sdk/downloads#interactive) to install gcloud
+
+### Enable the GCP IAM API
```
Comment (ttomsu) [inline setup/quickstart/halyard-gce.md:30]: The number of TLAs is too damn high :wink:

File: setup/quickstart/halyard-gce.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+    --filter="displayName:$GCS_SERVICE_ACCOUNT_NAME" \
+    --format='value(email)')
+
+# permission to create/modify buckets in your project
+gcloud projects add-iam-policy-binding $GCP_PROJECT \
+    --role roles/storage.admin \
+    --member serviceAccount:$GCS_SERVICE_ACCOUNT_EMAIL
+```
+
+```bash
```
Comment (ttomsu) [inline setup/quickstart/halyard-gce.md]: I'd add some text describing what this script is doing, just like what you did above with storage

File: setup/quickstart/halyard-gce.md [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+    --machine-type=n1-standard-4
+```
+
+SSH into the VM with the following ports forwarded as shown to allow access to
+the Spinnaker UI & API servers.
+
+> :warning: You need to SSH into the Halyard host VM from your local
+> workstation; SSHing from [Cloud Shell](https://cloud.google.com/shell/) or
+> another VM won't open the necessary SSH tunnels that will allow your local
+> web browser to access Spinnaker.
```
Comment (ttomsu) [inline setup/quickstart/halyard-gce.md:142]: Mention chromebooks also not supported for the same reason?

File: setup/quickstart/halyard-gce.md [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+
+```bash
+gcloud compute ssh $HALYARD_HOST \
+    --project=$GCP_PROJECT \
+    --zone=us-central1-f \
+    -- -L 9000:localhost:9000 -L 8084:localhost:8084
```
Comment (ttomsu) [inline setup/quickstart/halyard-gce.md:148]: extra `--`? Or maybe I'm just used to seeing it used with `--ssh-flag="..."`

File: setup/quickstart/halyard.md [hunk 5]
```diff
@@ -9,4 +9,5 @@ We've written guides for a few environments/configurations to get you started
 with Spinnaker by using Halyard. If your desired environment matches one of 
 those listed below, we recommend following the linked guide. 
 
-* [Deploy Spinnaker to GKE](/guides/tutorials/codelabs/halyard-getting-started)
+* [Halyard on GKE Quickstart](/setup/quickstart/halyard-gke/)
+* [Halyard on GCE Quickstart](/setup/quickstart/halyard-gce/)
```
Comment (ttomsu) [inline setup/quickstart/halyard.md]: Yay!

However, I think "GKE" is kind of an insider term - the product name is "Container Engine", so we should probably use that.

**Review by ttomsu — APPROVED** [review]

File: setup/quickstart/halyard-gce.md (hunk 1 above)
Comment (lwander) [reply setup/quickstart/halyard-gce.md:30]: heh

File: setup/quickstart/halyard-gce.md (hunk 4 above)
Comment (lwander) [reply setup/quickstart/halyard-gce.md:148]: This is the new arg passing style that gcloud is pushing us to, I think the `--ssh-flag` will be deprecated eventually

File: setup/quickstart/halyard.md (hunk 5 above)
Comment (lwander) [reply setup/quickstart/halyard.md]: gke

I think GKE is used quite often outside of Google, but I'll switch both to their full name.

**Pull request merged**